cpp/oneapi/dal/graph/BUILD

```diff
@@ -13,7 +13,20 @@ dal_module(
     ]
 )
 
+_DATA_DEPS = [
+    "@onedal//cpp/oneapi/dal/graph:test_data",
+]
+
 dal_test_suite(
-    name = "tests",
-    tests = [],
+    name = "graph_tests",
```

Comment:
The name should be "tests", the reason is that tests in `graph` submodule are discovered by the parent module using name "tests".